app/secrets.py

```diff
@@ -63,14 +60,12 @@ def as_jwk(self):
 
 
 class SecretStore:
-    def __init__(self, secrets):
-        self.secrets = secrets.get('secrets')
-        if 'keys' in secrets:
-            self.keys = {}
-            for _, kid in enumerate(secrets['keys']):
-                key = secrets['keys'][kid]
-                key_object = Key(kid, key['purpose'], key['type'], key['value'])
-                self.keys[kid] = key_object
+    def __init__(self, keys, secrets=None):
+        self.secrets = secrets.get('secrets') if secrets is not None else {}
+        self.keys = {}
```

Comment:
we should remove keys entirely from he secret store and have a KeyStore https://github.com/ONSdigital/sdc-cryptography/blob/master/sdc/crypto/key_store.py#L44